kafka-connect-neo4j/pom.xml

```
@@ -119,12 +123,15 @@
                             <ownerType>organization</ownerType>
                             <ownerName>Neo4j, Inc.</ownerName>
                             <ownerUrl>https://neo4j.com/</ownerUrl>
-                            <title>Neo4j Connector</title>
+                            <title>${connector.title}</title>
                             <documentationUrl>https://neo4j.com/docs/kafka/quickstart-connect/</documentationUrl>
-                            <description>It's a basic Apache Kafka Connect Neo4j Connector which allows moving data from Kafka topics into Neo4j via Cypher templated queries and vice versa.</description>
+                            <description>The connector runs on Apache Kafka Connect Clusters to sink data in Kafka
+                                topics into Neo4j and Aura databases and can use Neo4j/Aura as a source for Kafka topics
+                                via Cypher queries.
+                            </description>
                             <logo>assets/neo4j-logo.png</logo>
-                            <supportSummary><![CDATA[Support through <a href="https://neo4j.com/">Neo4j</a>]]></supportSummary>
-                            <sourceUrl>https://github.com/neo4j-contrib/neo4j-streams/tree/master/kafka-connect-neo4j</sourceUrl>
+                            <supportSummary>${connector.summary}</supportSummary>
+                            <sourceUrl>https://github.com/neo4j-contrib/neo4j-streams/tree/5.0</sourceUrl>
```

```suggestion
                            <sourceUrl>https://github.com/neo4j-contrib/neo4j-streams</sourceUrl>
```
do we need the branch in there?

---

yes, I think you're right - given that the default branch is now 5.0 - we can just omit the branch name.